yang-lsp/gradle/publishing.gradle

```diff
@@ -0,0 +1,109 @@
+java {
+	withJavadocJar()
+	withSourcesJar()
```

Comment:
Are these automatically generated? We currently have hand-written tasks for this in java-compiler-settings.gradle, maybe we can remove those?

Reply:
They are not needed here, I will remove it.